packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -190,9 +193,8 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   );
 
   // Maintain internal selected value state synced with `value` prop
-  const [selectedValue, setSelectedValue] = React.useState<T | undefined>(
-    value,
-  );
+  const [selectedValue, setSelectedValue] =
+    React.useState<T | undefined>(value);
```

Comment:
weird

Reply:
Yeah it must be from a different version of prettier. Idk why it didn't change in the monorepo PR